froide/foirequest/templates/foirequest/pdf/_foimessage.html

```diff
@@ -0,0 +1,42 @@
+{% load i18n %}
+<section>
+    <h2>{{ message.timestamp|date:"DATETIME_FORMAT" }} {{ message.real_sender }}</h2>
+    <dl>
```

Comment:
Maybe make this a table for easier readability?

Reply:
This gets rendered by weasyprint to PDF (no browser, no other CSS). The idea was to make it roughly look like Outlook‘s print style maybe. Not sure if table helps here.